### PR TITLE
Add Clams App

### PR DESCRIPTION
**Pull request by lnbc1QWFyb24**
This PR adds [Clams](https://clams.tech) to the supporting wallets list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,24 +7,24 @@ These are all the individual documents describing each small piece of protocol t
...
-| [04][04] | Auth base spec.                                             | [Alby][alby], [Balance of Satoshis][bos], [Blixt][blixt], [Breez][breez], [BlueWallet][bluewallet], [coinos][coinos], [Geyser][geyser], [LifPay][lifpay], [LNbits][lnbits], [@lntxbot][lntxbot], [LightningTipBot][ltb], [Phoenix][phoenix], [SeedAuth][seedauth], [SeedAuthExtension][sae], [SimpleBitcoinWallet][sbw], [Sparrow Wallet][sparrow], [ThunderHub][thunderhub], [Zap Desktop][zap], [Zeus][zeus] |
+| [03][03] | `withdrawRequest` base spec.                                | [Balance of Satoshis][bos], [Blixt][blixt], [BlueWallet][bluewallet], [Breez][breez], [Clams](clams), [CoinCorner][coincorner], [coinos][coinos], [Fountain][fountain], [LifPay][lifpay], [LNbits][lnbits], [@lntxbot][lntxbot], [LightningTipBot][ltb], [Muun][muun], [Phoenix][phoenix], [ShockWallet][shockwallet], [SimpleBitcoinWallet][sbw], [ThunderHub][thunderhub], [Wallet of Satoshi][wos], [Zap Android][zap], [Zap Desktop][zap], [Zap iOS][zap], [ZBD Discord][zbd], [ZBD Extension][zbd], [ZBD Telegram][zbd], [ZEBEDEE][zbd], [Zeus][zeus] |
```
Comment (hsjoberg) [inline README.md]: The links are currently broken. It must be done like this:
```suggestion
| [03][03] | `withdrawRequest` base spec.                                | [Balance of Satoshis][bos], [Blixt][blixt], [BlueWallet][bluewallet], [Breez][breez], [Clams][clams], [CoinCorner][coincorner], [coinos][coinos], [Fountain][fountain], [LifPay][lifpay], [LNbits][lnbits], [@lntxbot][lntxbot], [LightningTipBot][ltb], [Muun][muun], [Phoenix][phoenix], [ShockWallet][shockwallet], [SimpleBitcoinWallet][sbw], [ThunderHub][thunderhub], [Wallet of Satoshi][wos], [Zap Android][zap], [Zap Desktop][zap], [Zap iOS][zap], [ZBD Discord][zbd], [ZBD Extension][zbd], [ZBD Telegram][zbd], [ZEBEDEE][zbd], [Zeus][zeus] |
```

**Review by hsjoberg — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lnbc1QWFyb24) [reply README.md]: 🤦 Oops! That is now fixed.

**Comment by hsjoberg** [comment]
Great! LGTM. 💯

**Review by hsjoberg — APPROVED** [review]

**Comment by lnbc1QWFyb24** [comment]
Legend! Thanks for reviewing @hsjoberg!

**Pull request merged**